tests/resources/ModuleIntegrationSpec.cfc

```diff
@@ -0,0 +1,18 @@
+component extends="coldbox.system.testing.BaseTestCase" {
+
+    function beforeAll() {
+        super.beforeAll();
+
+        getController().getModuleService().registerAndActivateModule( 'stripe-cfml', 'testingModuleRoot' );
```

Comment:
Again, same question. I would prefer not to be renaming the module. Though, again, maybe there is something I don't understand about how this all works?